apps/view/src/RenderWorker/models.ts

```diff
@@ -0,0 +1,302 @@
+import commonPrefix from 'common-prefix'
+import randomColor from 'randomcolor'
+
+import vb from 'viewbox'
+import xid from '@tracespace/xml-id'
+import {Options as GerberToSvgOptions} from 'gerber-to-svg'
+
+// @ts-ignore: cannot figure out how to type this without breaking the import
+import clone from 'gerber-to-svg/clone'
+// @ts-ignore: cannot figure out how to type this without breaking the import
+import render from 'gerber-to-svg/render'
+
+import {Stackup, InputLayer, Options as StackupOptions} from 'pcb-stackup'
+
+import {
+  Optional,
+  BoardRender,
+  LayerRender,
+  LayersMap,
+  Board,
+  Layer,
+  LayerOptions,
+  BoardUpdate,
+} from '../types'
+
+import {svgToDataUri} from '../image'
+import {TYPE_DRILL} from '../layers'
+import {readFiles, fetchZipFile, writeFiles, FileStream} from './files'
+import {baseName} from './util'
+
+export type InputLayerFromFile = InputLayer & {gerber: FileStream}
+
+export type StackupFromFiles = Stackup<string, InputLayerFromFile>
+
+const DEFAULT_BOARD_NAME = 'untitled board'
+const RANDOM_ID_LENGTH = 20
+
+const SCALE_MM_TO_IN = 1 / 25.4
+const SCALE_IN_TO_MM = 25.4
+
+const getPcbStackup = import('pcb-stackup').then(m => m.default)
```

Comment:
I think I've unintentionally missed a `() =>` here. This still works, but the chunk is no longer loaded lazily; the import is started as soon as this file is run. I don't think that behavior is what I intended